app/renderer/components/navigation/navigator.js

```diff
@@ -150,6 +152,172 @@ class Navigator extends React.Component {
     ipc.off(messages.SHORTCUT_ACTIVE_FRAME_FORWARD, this.onForward)
   }
 
+  // BEM Level: navigator__menuBarAndNavigationBar__navigationBarWrapper
+  get navigationBarWrapper () {
```

Comment:
Thanks @bsclifton for digging in. I honestly am not sure the normal/proper way of handling them, so I would take the compromise you proposed :-)